job_creator.php

```diff
@@ -49,13 +47,20 @@ public function getInstallerVersion(string $githubRepository, string $branch): s
     public function createJob(int $phpIndex, array $opts): array
     {
         $installerKey = str_replace('.x-dev', '', $this->installerVersion);
+        $repo = explode('/', $this->githubRepository)[1];
+        if (in_array($repo, NO_INSTALLER_REPOS)) {
+            if (preg_match('#^[1-9]\.10$#', $this->getCleanedBranch())) {
+                $installerKey = '4.10';
```

Comment:
Aren't we trying to move away from hard-coded things like this? Maybe we need a `bool $ignoreNoInstallerRepo = false` argument to `getInstallerVersion()` instead, so we can just use the dynamic logic to find out what version of installer to use based on the branch name?